Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.7
+RUN apt-get update -qq && apt-get install -y
```

Comment:
What is the point of this line?

Reply:
That was to update all packages out of date; the intention is that it will be updating in every new build; other option could be to only update the security patches; as it is now is not slow so I opted with this one

Reply:
This is apparently considered an anti-pattern. The base docker image handles upgrades and security patched etc. so if you release often you will already have these. It makes sense then to defer that to the base image, which `FROM ruby:2.7` is probably deferring to.

Reply:
Ok, I remove it